frontend/src/components/studentInfo.tsx

```diff
@@ -0,0 +1,157 @@
+import useTranslation from "next-translate/useTranslation";
+import { useRouter } from "next/router";
+import apiPaths from "../properties/apiPaths";
+import useSWR from "swr";
+import { getAllStudentInfo } from "../api/calls/studentCalls";
+import { capitalize } from "../utility/stringUtil";
+import { Badge, Col, ListGroup, ListGroupItem, Row } from "react-bootstrap";
+import { SuggestionStrategy } from "../api/entities/SuggestionEntity";
+import { SuggestionModal } from "./suggestionModal";
+import { StudentStatus } from "./studentStatus";
+import { useSwrWithEdition } from "../hooks/utilHooks";
+
+export function StudentInfo() {
+    const { t } = useTranslation("common");
+    const router = useRouter();
+    const { id } = router.query as { id: string };
+
+    const { data, error } = useSwrWithEdition(apiPaths.students + "/" + id, getAllStudentInfo);
+
+    if (error || !data) {
+        return null;
+    }
+
+    let motivation;
+    if (data.student.motivationURI == "") {
```

Comment:
What if we have both? Or is this not possible?

Reply:
We should look in the studentform whether this is possible. Maybe we should ask Kasper?